homeassistant/components/wink.py

```diff
@@ -97,9 +97,16 @@ def setup(hass, config):
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN]['entities'] = []
     hass.data[DOMAIN]['unique_ids'] = []
+
+    def keep_alive_call(event):
+        """Call the Wink API endpoints to keep PubNub working."""
+        pywink.wink_api_fetch()
+        pywink.get_user()
+    hass.services.register(DOMAIN, 'keep_alive', keep_alive_call)
```

Comment:
If pubnubsubhandler cannot be trusted, let's move it indeed to `wink.py`. You can use [`async_track_time_interval`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/event.py#L168) to have it call automatically.

No need to expose it as a service then 👍